src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts

```diff
@@ -139,9 +139,9 @@ export class ReviewZoneWidget extends ZoneWidget implements ICommentThreadWidget
 		super(editor, { keepEditorSelection: true, isAccessible: true });
 		this._contextKeyService = contextKeyService.createScoped(this.domNode);
 
-		this._scopedInstantiationService = instantiationService.createChild(new ServiceCollection(
+		this._scopedInstantiationService = this._globalToDispose.add(instantiationService.createChild(new ServiceCollection(
```

Comment:
Adding comment to verify https://github.com/microsoft/vscode-pull-request-github/issues/5924

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion